src/test/java/org/itsallcode/openfasttrace/importer/legacytag/TestLegacyTagImporterFactory.java

```diff
@@ -92,8 +90,8 @@ public void testFactoryThrowsExceptionForUnsupportedFile()
     public void testFactoryCreatesImporterForSupportedFile() throws IOException
     {
         final File tempFile = this.temp.newFile();
-        final Importer importer = createImporter(asList(createConfig(tempFile.getAbsolutePath())),
-                tempFile.toPath());
+        final String glob = tempFile.getAbsolutePath().replace('\\', '/');
```

Comment:
Mind blown. :open_mouth: 
This is really still necessary in Java? Wow. I thought it was more robust cross-platform.